docs/book/src/cronjob-tutorial/testdata/project/Makefile

```
@@ -69,17 +69,32 @@ test: manifests generate fmt vet setup-envtest ## Run tests.
 # The default setup assumes Kind is pre-installed and builds/loads the Manager Docker image locally.
 # CertManager is installed by default; skip with:
 # - CERT_MANAGER_INSTALL_SKIP=true
+.PHONY: setup-test-e2e
+setup-test-e2e: ## Set up a Kind cluster for e2e tests if it does not exist
+	@command -v $(KIND) >/dev/null 2>&1 || { \
+		echo "Kind is not installed. Please install Kind manually."; \
+		exit 1; \
+	}
+	@$(KIND) get clusters | grep -q 'project-test-e2e' || { \
+		echo "Kind cluster not running, creating Kind cluster: project-test-e2e"; \
+		$(KIND) create cluster --name project-test-e2e --wait 5m; \
```

```suggestion
	$(KIND) create cluster --name $(KIND_CLUSTER_NAME)  --wait 5m; \
```

@kersten sorry I might not be very clear.
I think that
a) above we would need to have only the line to create the cluster ( not to check if it exist )
b) use an variable for we store the name of the cluster 


Then, after we run the tests

we call a target test-e2e-tweardown or cleanup something like that will delete the cluster name

---

For the envvar, I would rather wait for the PR #4617, or would you like me to merge the changes into this one?

---

Could we not do all here and keep a full proposal?

---

Sure, I will do it later. Probably tonight.